### ISSUE TITLE
Why Is the Loss Function of Large Models Cross-Entropy | 木鸟杂记

### ISSUE THREAD

**Issue by songpengwei** [songpengwei/blog-comment#133]
https://www.qtmuniao.com/en/2026/03/29/llm-loss/ 

 Prologue When first entering the world of large models, due to gaps in foundational math knowledge such as linear algebra, probability theory, and information theory, it’s easy to get lost among numer
